libs/commons/Testutil.ml

```diff
@@ -20,6 +27,4 @@ let mask_temp_paths ?depth ?replace () =
       ~tmpdir:(UFilename.get_temp_dir_name () |> UUnix.realpath)
       ()
   in
-  (* NOTE: Haven't investigated why yet, but order is important here,
-   *       we must mask the physical path before the original path. *)
   fun text -> text |> mask_physical_path |> mask_original_path
```

Comment:
maybe add a comment explaining why the order matters, maybe with an example showing
that if you do text |> mask_original_path |> mask_physical_path it does not work, even though you would think
those operations are communtative.
I think an example would clarify.

Otherwise LGTM.

Reply:
The order doesn't matter. It shouldn't matter.

`text |> mask_original_path |> mask_physical_path` used to not work if the original path was a substring of the physical path (as it is the case on some MacOS versions).

My point is that I see no reason to explain a problem that we don't have anymore and won't have again.